### PR TITLE
Update hu.m3u

### PR DESCRIPTION
**Pull request by kristoft13**
Added Zenebutik
Added Fix TV
Added Classics TV

### CODE REVIEW

File: streams/hu.m3u [hunk 1]
```diff
@@ -71,3 +71,9 @@ https://streaming.mytvback.com/stream/Nc8b6c6tUH4gh3GdRR-zFw/1617462698/channel0
 http://hitmusic.hu/hitmusic.m3u8
 #EXTINF:-1 tvg-id="" status="online",Hegyvidek Tv Budapest (576p) [Not 24/7]
 http://tv.hegyvidek.hu/stream_mpeg.flv
+#EXTINF:-1 tvg-id="" status="online",Zenebutik (720p) [Not 24/7]
+https://streaming.mytvback.com:443/stream/glkVRNvh_A6a2yUWuOYDBQ/1665263514/channel049/stream.m3u8
+#EXTINF:-1 tvg-id="" status="online",Fix TV (720p) [Not 24/7]
```
Comment (Alstruit) [inline streams/hu.m3u:76]:
```suggestion
#EXTINF:-1 tvg-id="FixTV.hu" status="online",Fix TV (720p) [Not 24/7]
```

File: streams/hu.m3u [hunk 2]
```diff
@@ -71,3 +71,9 @@ https://streaming.mytvback.com/stream/Nc8b6c6tUH4gh3GdRR-zFw/1617462698/channel0
 http://hitmusic.hu/hitmusic.m3u8
 #EXTINF:-1 tvg-id="" status="online",Hegyvidek Tv Budapest (576p) [Not 24/7]
 http://tv.hegyvidek.hu/stream_mpeg.flv
+#EXTINF:-1 tvg-id="" status="online",Zenebutik (720p) [Not 24/7]
```
Comment (Alstruit) [inline streams/hu.m3u:74]:
```suggestion
#EXTINF:-1 tvg-id="Zenebutik.hu" status="online",Zenebutik (720p) [Not 24/7]
```

**Review by Alstruit — CHANGES_REQUESTED** [review]
For adding Ids.

**Review by BellezaEmporium — DISMISSED** [review]

File: streams/hu.m3u [hunk 3]
```diff
@@ -71,3 +71,9 @@ https://streaming.mytvback.com/stream/Nc8b6c6tUH4gh3GdRR-zFw/1617462698/channel0
 http://hitmusic.hu/hitmusic.m3u8
 #EXTINF:-1 tvg-id="" status="online",Hegyvidek Tv Budapest (576p) [Not 24/7]
 http://tv.hegyvidek.hu/stream_mpeg.flv
+#EXTINF:-1 tvg-id="Zenebutik.hu" status="online",Zenebutik (720p) [Not 24/7]
+https://streaming.mytvback.com:443/stream/glkVRNvh_A6a2yUWuOYDBQ/1665263514/channel049/stream.m3u8
+#EXTINF:-1 tvg-id="FixTV.hu" status="online",Fix TV (720p) [Not 24/7]
+https://fixhd.tv:8082/fix/1080i/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Classics TV (720p) [Not 24/7]
+https://s03.diazol.hu:7152/hls/live.m3u8
```
Comment (LaneSh4d0w) [inline streams/hu.m3u:79]:
```suggestion
https://s03.diazol.hu:7152/live.m3u8
```
link above is simply a redirect to suggested link.

**Review by Alstruit — DISMISSED** [review]

**Review by Alstruit — APPROVED** [review]

**Review by what-a — APPROVED** [review]

**Pull request merged**